bolt/prices/v1/prices.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package bolt.prices.v1;
+
+import "bolt/assets/v1/assets.proto";
+import "google/protobuf/timestamp.proto";
+
+// PricesService: Prices venue service....
+service PricesService {
+  // Endpoint to fetch the price of the pair.
+  rpc GetPrice(GetPriceRequest) returns (GetPriceResponse);
+
+  // Endpoint to fetch multiple pairs prices in batch.
+  rpc GetPrices(GetPricesRequest) returns (GetPricesResponse);
+
+  // Endpoint to subscribe on the price changes.
+  rpc StreamPrice(StreamPriceRequest) returns (stream StreamPriceResponse);
```

Comment:
Not sure if this is needed as `StreamPrices` can be used with same expected behaviour.

Reply:
I agree, but I did intend to keep the parity with messages on repo at this point.

Reply:
The current sui-outpost has only one `stream_prices` 

https://github.com/phi-labs-ltd/sui-outpost/blob/37ed83c7574431756e21bc813453dd8887bb0e83/offchain/rust/grpc/proto-types/src/proto/bolt.prices.v1.tonic.rs#L137

Reply:
Ahh, you are right! I probably just rushed it seeing separate request/response types. Let's me fix it then.